pymc/sampling/mcmc.py

```diff
@@ -612,14 +615,9 @@ def sample(
         idata = pm.to_inference_data(mtrace, **ikwargs)
 
         if compute_convergence_checks:
-            if draws - tune < 100:
-                warnings.warn(
-                    "The number of samples is too small to check convergence reliably.",
-                    stacklevel=2,
-                )
```

Comment:
This is now checked by `run_convergence_checks`, just like it already checked for a minimum number of chains